Downloading requirements

I was trying to install deeppavlov and had a problem following the installation steps.

1) There is no download.py file in root folder, it is in `deeppavlov/download.py`
``` sh
python download.py [-all] 
```

2) Even if I use that file it outputs the error:
``` sh
(env) root@mysexyhost:~/work/ipavlov/DeepPavlov# python3 deeppavlov/download.py
/home/ubuntu/work/ipavlov/env/local/lib/python3.5/site-packages/h5py/__init__.py:36: FutureWarning: Conversion of the second argument of issubdtype from `float` to `np.floating` is deprecated. In future, it will be treated as `np.float64 == np.dtype(float).type`.
  from ._conv import register_converters as _register_converters
Using TensorFlow backend.
2018-03-12 07:34:11.490 ERROR in 'deeppavlov.core.models.serializable'['log'] at line 54: LOGGER ERROR: Can not initialise deeppavlov.core.models.serializable logger, logging to the stderr. Error traceback:
Traceback (most recent call last):
  File "/home/ubuntu/work/ipavlov/DeepPavlov/deeppavlov/core/common/log.py", line 32, in get_logger
    with open(log_config_path) as log_config_json:
TypeError: invalid file: PosixPath('/home/ubuntu/work/ipavlov/DeepPavlov/deeppavlov/log_config.json')
2018-03-12 07:34:11.491 ERROR in 'deeppavlov.core.models.keras_model'['log'] at line 54: LOGGER ERROR: Can not initialise deeppavlov.core.models.keras_model logger, logging to the stderr. Error traceback:
Traceback (most recent call last):
  File "/home/ubuntu/work/ipavlov/DeepPavlov/deeppavlov/core/common/log.py", line 32, in get_logger
    with open(log_config_path) as log_config_json:
TypeError: invalid file: PosixPath('/home/ubuntu/work/ipavlov/DeepPavlov/deeppavlov/log_config.json')
Traceback (most recent call last):
  File "deeppavlov/download.py", line 24, in <module>
    from deeppavlov.core.data.utils import download, download_decompress
  File "/home/ubuntu/work/ipavlov/DeepPavlov/deeppavlov/__init__.py", line 1, in <module>
    import deeppavlov.core.models.keras_model
  File "/home/ubuntu/work/ipavlov/DeepPavlov/deeppavlov/core/models/keras_model.py", line 39, in <module>
    class KerasModel(NNModel, metaclass=TfModelMeta):
  File "/home/ubuntu/work/ipavlov/DeepPavlov/deeppavlov/core/models/keras_model.py", line 143, in KerasModel
    sample_weight_mode=None, weighted_metrics=None, target_tensors=None):
  File "/home/ubuntu/work/ipavlov/env/local/lib/python3.5/site-packages/overrides/overrides.py", line 70, in overrides
    method.__name__)
AssertionError: No super class method found for "load"
```

--- a/README.md
+++ b/README.md
@@ -38,15 +38,15 @@ View video demo of deployment of a goal-oriented bot and a slot-filling model wi
           
  * Run goal-oriented bot with Telegram interface:
  ```
- python deeppavlov/deep.py interactbot deeppavlov/configs/go_bot/gobot_dstc2.json -t <TELEGRAM_TOKEN>
+ python -m deeppavlov.deep interactbot deeppavlov/configs/go_bot/gobot_dstc2.json -t <TELEGRAM_TOKEN>
  ```
  * Run goal-oriented bot with console interface:
  ```
- python deeppavlov/deep.py interact deeppavlov/configs/go_bot/gobot_dstc2.json
+ python -m deeppavlov.deep interact deeppavlov/configs/go_bot/gobot_dstc2.json
  ```
  * Run slot-filling model with Telegram interface:
  ```
- python deeppavlov/deep.py interactbot deeppavlov/configs/ner/slotfill_dstc2.json -t <TELEGRAM_TOKEN>
+ python -m deeppavlov.deep interactbot deeppavlov/configs/ner/slotfill_dstc2.json -t <TELEGRAM_TOKEN>
  ```
  * Run slot-filling model with console interface:
  ```
@@ -128,7 +128,7 @@ DeepPavlov is built on top of machine learning frameworks (TensorFlow, Keras). O
 
 To use our pre-trained models, you should first download them:
 ```
-python deeppavlov/download.py [-all] 
+python -m deeppavlov.download [-all] 
 ```
 * running this command without options will download basic examples, `[-all]` option will download **all** our pre-trained models.
 * Warning! `[-all]` requires about 10 GB of free space on disk.
@@ -136,7 +136,7 @@ python deeppavlov/download.py [-all]
 Then you can interact with the models or train them with the following command:
 
 ```
-python deeppavlov/deep.py <mode> <path_to_config>
+python -m deeppavlov.deep <mode> <path_to_config>
 ```
 
 * `<mode>` can be 'train', 'interact' or 'interactbot'

--- a/README.md
+++ b/README.md
@@ -128,7 +128,7 @@ DeepPavlov is built on top of machine learning frameworks (TensorFlow, Keras). O
 
 To use our pre-trained models, you should first download them:
 ```
-python download.py [-all] 
+python deeppavlov/download.py [-all] 
 ```
 * running this command without options will download basic examples, `[-all]` option will download **all** our pre-trained models.
 * Warning! `[-all]` requires about 10 GB of free space on disk.
@@ -136,7 +136,7 @@ python download.py [-all]
 Then you can interact with the models or train them with the following command:
 
 ```
-python deep.py <mode> <path_to_config>
+python deeppavlov/deep.py <mode> <path_to_config>
 ```
 
 * `<mode>` can be 'train', 'interact' or 'interactbot'
@@ -147,13 +147,13 @@ For 'interactbot' mode you should specify Telegram bot token in `-t` parameter o
 
 Available model configs are:
 
-*configs/go_bot/gobot_dstc2.json*
+*deeppavlov/configs/go_bot/gobot_dstc2.json*
 
-*configs/intents/intents_dstc2.json*
+*deeppavlov/configs/intents/intents_dstc2.json*
 
-*configs/ner/slotfill_dstc2.json*
+*deeppavlov/configs/ner/slotfill_dstc2.json*
 
-*configs/error_model/brillmoore_wikitypos_en.json*
+*deeppavlov/configs/error_model/brillmoore_wikitypos_en.json*
 
 ---
 

--- a/README.md
+++ b/README.md
@@ -38,19 +38,19 @@ View video demo of deployment of a goal-oriented bot and a slot-filling model wi
           
  * Run goal-oriented bot with Telegram interface:
  ```
- python deep.py interactbot configs/go_bot/gobot_dstc2.json -t <TELEGRAM_TOKEN>
+ python deeppavlov/deep.py interactbot deeppavlov/configs/go_bot/gobot_dstc2.json -t <TELEGRAM_TOKEN>
  ```
  * Run goal-oriented bot with console interface:
  ```
- python deep.py interact configs/go_bot/gobot_dstc2.json
+ python deeppavlov/deep.py interact deeppavlov/configs/go_bot/gobot_dstc2.json
  ```
  * Run slot-filling model with Telegram interface:
  ```
- python deep.py interactbot configs/ner/slotfill_dstc2.json -t <TELEGRAM_TOKEN>
+ python deeppavlov/deep.py interactbot deeppavlov/configs/ner/slotfill_dstc2.json -t <TELEGRAM_TOKEN>
  ```
  * Run slot-filling model with console interface:
  ```
- python deep.py interact configs/ner/slotfill_dstc2.json
+ python -m deeppavlov.deep interact deeppavlov/configs/ner/slotfill_dstc2.json
  ```
 ## Conceptual overview
 

